### DISCUSSION TITLE
Let the user know there are other recordings available

### DISCUSSION THREAD

**Discussion by Aquathing** [blakeblackshear/frigate#13356] [Ideas]
They other day I was trying to view a recording 24+ hours in the past, and couldn't find it.
This made me question my config, but everything seemed perfect, so I couldn't really understand what was going on and was about to open an issue until I found this: https://github.com/blakeblackshear/frigate/discussions/12932

Given that there are technical reasons for this limitation, perhaps it'd be good to tell the user with a button that they can go back in time from within the timelime.

![image](https://github.com/user-attachments/assets/106c797c-ee46-4ee8-9e05-b6e763b07d3e)
Something like this, sorry but I'm bad at CSS.

**Comment by hawkeye217** [comment]
The timeline in the Review pane and in History view displays 24 hours, so this is working as designed. The calendar filter shows small dots for unreviewed items in the Review pane, and the calendar filter can be used in History view to change the current day being viewed.

**Reply by Aquathing** [reply]
> The timeline in the Review pane and in History view displays 24 hours, so this is working as designed.

Absolutely! What I meant is that if it is your first time using the live view for older recordings, it might be hard for you to notice that the calendar is actively filtering, especially when you are on mobile where the calendar is hidden behind the settings button.

That's why I suggested to let the user know that at the bottom of the timeline.

**Reply by hawkeye217** [reply]
Thanks for the suggestion!

**Reply by Aquathing** [reply]
Thank you for the great software :smile:
